test/util/rs-factory.js

```diff
@@ -0,0 +1,16 @@
+'use strict'
+
+var xtend = require('xtend')
+
+module.exports = function (testCommon) {
+  return function createReadStream (db, options) {
+    if (!testCommon.encodings) {
+      options = xtend(options, {
+        keyAsBuffer: false,
+        valueAsBuffer: false
+      })
```

Comment:
The tests implicitly relied on the default utf8 encoding, for the db to return strings. When the db isn't wrapped with `encoding-down`, `*asBuffer: false` achieves the same.